images/node/Dockerfile.product

```diff
@@ -0,0 +1,39 @@
+FROM rhel7
```

Comment:
Same comments about style - you should be minimizing the number of layers by combining declarations.